controllers/awsadapterconfig_controller.go

```diff
@@ -253,9 +253,9 @@ func (r *AWSAdapterConfigReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	}
 
 	if x, err := eksClient.ListNodegroups(context.TODO(), &eks.ListNodegroupsInput{ClusterName: objOld.Spec.Name}); err == nil {
+		objNew.Status.EKSCluster.Compute.NodeGroups = []*securityv1alpha1.EKSNodeGroup{}
```

Comment:
There was no need to move the code here right? Statememt has same effect even it had not been changed, right?

Reply:
If not moved outside the for loop, the `nodeGroups` will be initialized to an empty array for each nodegroup element, in turn always making `nodeGroups` a single element array irrespective of how many nodegroups we have in our cluster.

Reply:
Got it. Could not figure out indentation easily.